datamodel/include/datamodel/visual/texture_types.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef RCBE_ENGINE_TEXTURE_TYPES_HPP
+#define RCBE_ENGINE_TEXTURE_TYPES_HPP
+
+#include <GL/gl.h>
+
+namespace rcbe::visual {
+enum class GLTextureWrappingType {
```

Comment:
Leave for this one and other marked `GL` enums, TODOs dennoting that theses should be renamed after more rendererers are introduced.